types/node/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     ],
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es6",
+        "target": "esnext",
```

Comment:
What’s this for?

Reply:
Been a moment but I believe this is to fix bigint support in tests.